### ISSUE TITLE
The rule book does not contain the train roster

### ISSUE THREAD

**Issue by gloinul** [akvik/18Norway#21]
So the rule book does not contain the train roster. I think it should so that one can verify that one have all the trains, or when reading it and not wanting to open the box to find a company charter.

**Comment by akvik** [comment]
Yes, good point, as you say it's on the charter, it's also on the phase chart, but it will make sense to include it in the rules, too.
